render/detailed/connections.go

```diff
@@ -85,25 +85,111 @@ func (row connection) ID() string {
 	return fmt.Sprintf("%s:%s-%s:%s-%s", row.remoteNodeID, row.remoteAddr, row.localNodeID, row.localAddr, row.port)
 }
 
+type connections struct {
```

Comment:
`connection`, `connections` and `[]Connection` make the code a a bit hard to read. I would consider renaming this type to something more explicit like `connectionCounts`